src/Cvl/LDAPWrapper/LDAPWrapper.php

```diff
@@ -459,6 +459,22 @@ public function isAdmin($userDN) {
 	}
 
 	/**
+	 * @param $userDN
+	 * @return bool whether specified user is disabled or not (works only with ActiveDirectory, otherwise returns false)
+	 */
+	public function isDisabled($userDN) {
```

Comment:
Perhaps name isUserDisabled, since it cannot be called for group DNs, and it's just a string so can't rely on some type hinting to imply the usage.